assets/appbehattests/app.feature

```diff
@@ -0,0 +1,8 @@
+@app @moodledocker
+Feature: The app starts in behat
+  In order to test that the app it working properly
+  As moodle-docker
+  I need a way to verify this
+
+  Scenario: Start the app
+    Given I entered the app

```

Comment:
I think this step doesn't exist, and I would add some assertion just to make it clear (it's weird to have tests without assertions).

```suggestion
    When I launch the app runtime
    Then I should find "Welcome to the Moodle App!" in the app
```

Reply:
I have some fixes for it I'm going to use a course and mod_label so that we can guarantee the text we want will be there.